src/pages/studyView/charts/barChart/BarChart.tsx

```diff
@@ -203,6 +203,7 @@ export default class BarChart extends React.Component<IBarChartProps, {}> implem
                     />
                     <VictoryBar
                         barRatio={this.barRatio}
+                        labelComponent={<VictoryTooltip/>}
```

Comment:
To keep the style inline, try to use DefaultTooltip instead of VictoryTooltip.

Reply:
I don't think DefaultToolTip will work here unfortunately.  You should study how we hand tooltip in src/pages/resultsView/cancerSummary/CancerSummaryChart.tsx.   It's unfortunately kind of complicated, but that's what's necessary to achieve HTML tooltip in context of Victory SVG.

Reply:
Yes exactly, DefaultToolTip is not working here that's why I added VictoryTooltip.We can define a different template of VictoryTooltip for Styling purpose.so that Style Remains the same. Can we?

Reply:
Hi, no, please use the DefaultToolTip as in the example I gave.  It's more work but we need to have tooltip that supports HTML.  That is critical for most use cases of our tooltips.